toolchain/symbiflow-toolchain/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+mkdir -p $PREFIX
+
+curl -s https://storage.googleapis.com/symbiflow-arch-defs-gha/symbiflow-toolchain-latest | xargs wget -qO- | tar -xJC $PREFIX
```

Comment:
We **really** need to make sure that there is a connection between these assets and the symbiflow-arch-defs git revision and not use the random latest version....

Reply:
So this needs to download the results which match the git revision used to generate the package metadata. That info should be available in the shell environment somewhere...